### ISSUE TITLE
Enable Mapillary layer when being referred from Mapillary site

### ISSUE THREAD

**Issue by PolyglotOpenstreetmap** [openstreetmap/iD#2524]
Hi,

When one clicks open in iD on the Mapillary site, iD opens fine. The offset is a bit off, but what is worse is that no images are visible, as the Mapillary is switched off.

It gets old really fast to have to enable it over and over again for each picture one wants to add something to OSM for with help of iD.

Jo

**Comment by PolyglotOpenstreetmap** [comment]
This is the issue in the Mapillary repo:

https://github.com/mapillary/mapillary_issues/issues/586

**Comment by PolyglotOpenstreetmap** [comment]
No comments whatsoever. I was wondering whether to start a new ticket for this related problem. When I'm  coming from HOT tasking manager:
http://tasks.hotosm.org/project/870#task/249

I also want to see the Mapillary pictures. I'll stop bugging you guys with this as soon as we have support for Mapillary in JOSM, but it would still be nice to solve this.

What I wanted to to is to make people aware of the presence of these picture sequences. But If I'm going to have to enable the Mapillary layer each and every time, I open a square, I'll just as easily give up in despair. One step in the right direction could be to remember my preference to see those pictures, but that would still not help your target audience, who may never figure out how to enable viewing them.

Jo

Jo

**Comment by bhousel** [comment]
> One step in the right direction could be to remember my preference to see those pictures, but that would still not help your target audience, who may never figure out how to enable viewing them.

Yes, I think it's a good idea for us to allow enabling of the various map data overlays through the url hash api, so that users pushed to iD can see the correct layers enabled. (similar to how we have a background url parameter)
